### PR TITLE
Use the brand new issue template

### PR DESCRIPTION
**Pull request by ksco**
The result is exactly the same as before, but it provides a way more nicer form, see the preview here: https://github.com/ksco/box86-compatibility-list/issues/new/choose

I bet @ricoazzurro will like it.

### CODE REVIEW

**Comment by ricoazzurro** [comment]
looks good.
I'd like two functions if you have time :)
1. some times I copy and past a large bulk of contents at once, if I use the new form, I'll copy/past multiple times.
so a button/link to switch will be nice.
2. I usually test like 10 games together, when I have time, so most of the info (version, set-up, etc) will be the same,
is save-function possible? hmmm, I may use auto-fill function from my chrome :)

**Comment by ricoazzurro** [comment]
I'm a merciless testing robot with no feeling, muahahahahahaha
--
I've finished this round of game testing.
And plan to read the git book zzzzZZZZZ
--
Thank you for the new option of issue template.
Have a lovely weekend <3

**Review by mcagabe19 — APPROVED** [review]
I also agree ksco

**Comment by ksco** [comment]
1. Well, if you want copy-paste, you can always use the blank template: https://github.com/ptitSeb/box86-compatibility-list/issues/new/
2. Yeah, auto-fill is the way to go.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```
Comment (ksco) [inline .github/ISSUE_TEMPLATE/config.yml:1]: @ptitSeb Oh this has to be set to true if submitting an issue through blank template should be an option, it's up to you...

Comment (ptitSeb) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Let's leave like that for now. Most ticket are formated well enough for now.

**Pull request merged**